PRASFiles.jl/src/Results/write.jl

```diff
@@ -0,0 +1,130 @@
+function generate_systemresult(results::R, pras_sys::SystemModel) where {R <: Tuple{Vararg{Result}}}
+ 
+    shortfall_idx = findfirst(name.(typeof.(results)) .== ShortfallResult)
+    shortfall = results[shortfall_idx]
+    
+    surplus_idx = findfirst(name.(typeof.(results)) .== SurplusResult)
```

Comment:
How important is it to store these other result types? Most of the complexity/messiness here comes from that. Unless we really need it I'd rather just stick to exporting RA metrics